src/components/BlogArticle/index.js

```diff
@@ -20,18 +21,37 @@ const BlogArticle = ({
     const { t } = useTranslation(['common', 'blog'])
     const defaultTitle = 'Blog - Instant Visio'
 
+    const blogArticle = useRef(null)
+
     useDocumentTitle(
         pageTitle ? `${pageTitle} - ${defaultTitle}` : defaultTitle
     )
 
+    useEffect(() => {
+        Array.from(blogArticle.current.querySelector('span').children).map((el) => {
```

Comment:
this could have been done in another way (to prevent writing css directly on the element) but I'll accept the PR though

Reply:
Sure. With the state I guess